sigs.yaml

```
@@ -104,6 +104,39 @@ sigs:
             - https://raw.githubusercontent.com/kubevirt/hostpath-provisioner-operator/main/OWNERS
             - https://raw.githubusercontent.com/kubevirt/hyperconverged-cluster-operator/master/OWNERS
             - https://raw.githubusercontent.com/kubevirt/must-gather/main/OWNERS
+    - dir: sig-ci
+      name: KubeVirt CI Operations Group
+      mission_statement: |
+        Ensure that the CI infrastructure is operational, that CI jobs are executed in a timely manner and PRs of any of the onboarded projects are not blocked.
+      leadership:
+        chairs:
+        - github: brianmcarey
+          name: "Brian Carey"
+          company: Red Hat
+        - github: dhiller
+          name: "Daniel Hiller"
+          company: Red Hat
+      meetings:
+      - description: KubeVirt CI Taskforce Sync meeting
+        day: Monday
+        time: "09:00"
+        tz: UTC
+        frequency: weekly
+        url: https://meet.google.com/pcy-dnin-ojj
```

```suggestion
        url: https://zoom.us/j/96583958896
```
I've set this to an hour. Please let me know if it should be shorter (or longer..?)